### PR TITLE
move countries from item to content

### PR DESCRIPTION
**Pull request by jamiesprax**

### CODE REVIEW

File: src/main/java/org/atlasapi/media/entity/simple/Playlist.java [hunk 1]
```diff
@@ -33,6 +34,7 @@
     private Set<ContentIdentifier> availableContent = Sets.newHashSet();
     private Set<ContentIdentifier> recentContent = Sets.newHashSet();
     private List<SeriesIdentifier> series = Lists.newArrayList();
+    private Set<Country> countriesOfOrigin = Sets.newHashSet();
```
Comment (dragospitica) [inline src/main/java/org/atlasapi/media/entity/simple/Playlist.java]: `Brand, Series and Item` seem to be derived from `Content`. Why are countries needed here?

File: src/main/java/org/atlasapi/media/entity/Content.java [hunk 2]
```diff
@@ -186,6 +196,7 @@ public static void copyTo(Content from, Content to) {
         to.genericDescription = from.genericDescription;
         to.similarContent = from.similarContent;
         to.events = ImmutableList.copyOf(Iterables.transform(from.events, EventRef.COPY));
+        to.countriesOfOrigin = Sets.newHashSet(from.countriesOfOrigin);
```
Comment (tristanbeardmore-gray) [inline src/main/java/org/atlasapi/media/entity/Content.java]: Should this be an ImmutableSet?

File: src/main/java/org/atlasapi/media/entity/Content.java [hunk 3]
```diff
@@ -42,6 +43,7 @@
     private Boolean genericDescription;
     private ImmutableList<EventRef> events = ImmutableList.of();
     private Integer editorialPriority;
+    private Set<Country> countriesOfOrigin = Sets.newHashSet();
```
Comment (tristanbeardmore-gray) [inline src/main/java/org/atlasapi/media/entity/Content.java]: ImmutableSet?

File: src/main/java/org/atlasapi/media/entity/simple/Playlist.java (hunk 1 above)
Comment (jamiesprax) [reply src/main/java/org/atlasapi/media/entity/simple/Playlist.java]: Its the simple model

**Pull request merged**